scripts/install.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+#
+# origin https://github.com/al-one/hass-xiaomi-miot/blob/master/install.sh
+#
+# wget -q -O - https://raw.githubusercontent.com/wuwentao/midea_ac_lan/master/scripts/install.sh | bash -
+# wget -q -O - https://raw.githubusercontent.com/wuwentao/midea_ac_lan/master/scripts/install.sh | ARCHIVE_TAG=v0.4.2 bash -
```

Comment:
I would first download `manifest.json` and then take the version from there to avoid more maintenance.